### ISSUE TITLE
Unrecognized functions used outside of CSS declaration groups are not caught by static checks

### ISSUE THREAD

**Issue by iflan** [google/closure-stylesheets#38]
Originally reported on Google Code with ID 19
```
Example: 

/* Does not generate error. */
@def RANDOM_COLOR randomColor('light');


/* Correctly generates error: Unknown function "randomColor". */
button {
  color: randomColor('light');
}
```

Reported by `cpeisert` on 2012-05-29 02:48:58

**Comment by iflan** [comment]
@cpeisert, you are correct. However this should give you an error:
```css
@def RANDOM_COLOR randomColor('light');

button {
   background-color: RANDOM_COLOR;
}
```
The reason is that the contents of `@def`s aren't checked until they are used.

Please re-open if this is not the case.
